### ISSUE TITLE
account rating needs to come from AccountNode statistic

### ISSUE THREAD

**Issue by scd** [theCrag/website#2537]

**Comment by scd** [comment]
I have partially completed this so that accounts now get the following template data:

```
"rating": {

    "sport": {
        "decay": 1593,
        "total": 3164
    },
    "trad": {
        "decay": 3483,
        "total": 3484
    },
    "boulder": {
        "decay": 2208,
        "total": 2420
    }
},
```

@brendanheywood somewhere else you had some suggestions on how to get the users default rating system for each of the CPR types so I can convert to a grade.

I am thinking that we use the users ticks to determine this. Group ascents by Style and RatingSystem then and find the maximum count.

If there are no rating systems then this probably means there are no ticks and no score. 

Does this align with your thinking?

**Comment by scd** [comment]
See also #2505

**Comment by brendanheywood** [comment]
Yes I've made a distinct issue #2526 to cover the algorithm for choosing a grade context for an account.

Also unless there is hassle it might be better making this template chunk and the chunk in #2505 identical?

An idle thought I just had: the relationship between the total and decay values says a lot about a person. Even if I have a small boulder total but it is close to my decay, then that means I am actively bouldering. If my trad decay is low and my total is high it means I used to be a trad climber. I think this may end up being a much better heuristic for deciding what kind of climber I am *right now*
